include/stl2/detail/functional/arithmetic/minus.hpp

```diff
@@ -0,0 +1,63 @@
+// cmcstl2 - A concept-enabled C++ standard library
+//
+//  Copyright Casey Carter
+//  Copyright Christopher Di Bella
+//
+//  Use, modification and distribution is subject to the
+//  Boost Software License, Version 1.0. (See accompanying
+//  file LICENSE_1_0.txt or copy at
+//  http://www.boost.org/LICENSE_1_0.txt)
+//
+// Project home: https://github.com/caseycarter/cmcstl2
+//
+#ifndef STL2_DETAIL_FUNCTIONAL_ARITHMETIC_MINUS_HPP
+#define STL2_DETAIL_FUNCTIONAL_ARITHMETIC_MINUS_HPP
+
+#include <stl2/detail/concepts/core.hpp>
+#include <stl2/detail/functional/arithmetic/plus.hpp>
+#include <stl2/detail/fwd.hpp>
+#include <stl2/detail/numeric_traits/identity.hpp>
+#include <type_traits>
+#include <utility>
+
+STL2_OPEN_NAMESPACE {
+	namespace ext {
+		template<class T, class U>
+		META_CONCEPT __differenceable_with = __summable_with<T, U> &&
+			requires(__uncvref<T> t, __uncvref<U> u) {
+				{ +std::forward<T>(t) } -> Same<T>;
+				{ -std::forward<T>(t) } -> Same<T>;
+				{ +std::forward<U>(u) } -> Same<U>;
+				{ -std::forward<U>(u) } -> Same<U>;
```

Comment:
I _think_ these are supposed to be `Common`, not `Same`. I should add tests for the arithmetic operations.

Reply:
Yep, test for `partial_sum` reveals that this is the case.

Reply:
Fixed in cd3992c3f27f90797128f90764839b610f5fe088, which also adds `negate`.